### PR TITLE
fix: correct Expenses Test assertion

### PR DESCRIPTION
**Pull request by snitin315**
Resolve [#3032](https://github.com/opencollective/opencollective/issues/3032)

# Description

Fix Expense test.

# Screenshots

NA

### CODE REVIEW

File: components/expenses/__tests__/Expenses.test.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ describe('Expenses component', () => {
       expect(component.find('[data-cy="mark-expense-as-paid-btn"]')).not.toBeFalsy();
 
       // make sure none are disabled
-      expect(component.find('[data-cy="expense-actions"] button[disabled]').lenght).toEqual(undefined);
+      expect(component.find('[data-cy="expense-actions"] button[disabled]').length).toEqual(4);
```
Comment (znarf) [inline components/expenses/__tests__/Expenses.test.js:83]: Looks like this is not matching the comment. Not sure what to do.

Comment (snitin315) [reply components/expenses/__tests__/Expenses.test.js:83]: Maybe the code is incorrect?

Comment (Betree) [reply components/expenses/__tests__/Expenses.test.js:83]: I'm tempted to discard this line, it's better to have nothing than a false assumption. Probably not worth digging.

Comment (snitin315) [reply components/expenses/__tests__/Expenses.test.js:83]: I will remove it :+1:

**Pull request closed without merging**